endoreg_db/views/anonymization_overview.py

```
@@ -7,16 +7,64 @@
+
+class AnonymizationOverviewView(ListAPIView):
+    """
+    GET /api/anonymization/items/overview/
+    --------------------------------------
+    Returns a flat list (Video + PDF) ordered by newest upload first.
+    """
+    serializer_class = FileOverviewSerializer
+    permission_classes = DEBUG_PERMISSIONS   
+    pagination_class = NoPagination
+
+class AnonymizationOverviewView(ListAPIView):
```

Ruff (0.12.2)

Redefinition of unused `AnonymizationOverviewView` from line 26

(F811)